EventPlanning.BL/EventManager.cs

```diff
@@ -175,6 +175,47 @@ public bool PatchEvent(EventData eventData, int eventId)
             return repository.Update(ev);
         }
 
+        /// <summary>
+        /// Adds activities to event
+        /// </summary>
+        /// <param name="activityData"></param>
+        /// <param name="eventId"></param>
+        /// <returns></returns>
+        public bool AddActivitiesToEvent(List<ActivityData> activityData, int eventId)
+        {
+            var ev = GetEvent(eventId);
+
+            if (ev == null)
+            {
+                return false;
+            }
+
+            var ea = GetEventActivities(eventId);
+            var newEventActivities = new List<EventActivity>();
+
+            foreach (var activity in activityData)
+            {
+                if (!ea.Exists(x => x.Title == activity.Title))
```

Comment:
If activity does not exist, why not add it?
If we have some random game called "Pandemic with extra stuff" why not add it?